### PR TITLE
Update Cloud Images control panel links to use https and clarify lang…

### PR DESCRIPTION
**Pull request by RackerKate**
…uage.

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f02f7da084/how-to/).

**Comment by RackerKate** [comment]
@cyrichardson For your review please. Sorry to bombard you with these.

**Comment by cyrichardson** [comment]
@RackerKate No worries. Will add this to my queue.

**Comment by RackerKate** [comment]
@cyrichardson Please review - OK to go now.

File: content/cloud-images/transferring-images-between-regions-of-the-rackspace-open-cloud.md [hunk 1]
```diff
@@ -74,21 +74,24 @@ To work in the source region, use the following steps:
...
 
 3.  You need a container in your Cloud Files account to put the exported
     image in. If you don't already have one, you can create a container
     by using the Cloud Control Panel by using the following steps:
 
-    1. In the top navigation bar, click **Storage > Files**.
-    2. Click **Create Container**.
-    3. Specify a name, the source region, and the container type.
-    4. Click **Create Container**.
+    1. In the top navigation bar, click **Select a Product > Rackspace
```
Comment (cyrichardson) [inline content/cloud-images/transferring-images-between-regions-of-the-rackspace-open-cloud.md]: Do you want to start this list of steps with the "Log in to the [Cloud Control Panel](https://login.rackspace.com)." one??

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-b4c5f0e44e/how-to/)

File: content/cloud-images/transferring-images-between-regions-of-the-rackspace-open-cloud.md [hunk 2]
```diff
@@ -74,21 +74,25 @@ To work in the source region, use the following steps:
...
 3.  You need a container in your Cloud Files account to put the exported
     image in. If you don't already have one, you can create a container
     by using the Cloud Control Panel by using the following steps:
 
-    1. In the top navigation bar, click **Storage > Files**.
-    2. Click **Create Container**.
-    3. Specify a name, the source region, and the container type.
-    4. Click **Create Container**.
+    1. Log in to the [Cloud Control Panel](https://login.rackspace.com).
+    1. In the top navigation bar, click **Select a Product > Rackspace
```
Comment (catlook) [inline content/cloud-images/transferring-images-between-regions-of-the-rackspace-open-cloud.md]: You have an extra #1.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-fa581d8bb2/how-to/).

**Pull request merged**